package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "headquarters-node",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
It's a bug introduced by a change in the API, but only because it's a totally new API itself

There are two ways to look at this in my opinion:

1. Assuming this would be a publicly used API, with a proper release cycle.
In that case, `hq.groupbuddies.com` would still exist, with the old version & database.
In that case, the `0.1` version of this package should still use that endpoint to keep compatibility, and a `0.2` version would be released to point to the new one.
`0.1.1` wouldn't be a good solution because any package depending on `~> 0.1.0` would still evaluate to `0.1.1`, and it would end up using the new endpoint, which is probably not the desired result

2. Assuming this is a private API, used only by us. And if we assume that, I'd say this discussion is a waste of time :stuck_out_tongue: